### PR TITLE
Feature/add modal component/126036281

### PR DESCRIPTION
**Pull request by alexisBimatovs**
https://www.pivotaltracker.com/story/show/126036281

screen shot 2016-07-15 at 1 28 51 pm
screen shot 2016-07-15 at 1 29 01 pm
screen shot 2016-07-15 at 1 29 10 pm
screen shot 2016-07-15 at 2 02 04 pm

### CODE REVIEW

File: lib/forever_style_guide/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-module ForeverStyleGuide
-  VERSION = "3.0.7"
+module ForeverStyleGuide
```
Comment (schweitzdesign) [inline lib/forever_style_guide/version.rb]: whitespace

File: app/assets/stylesheets/forever_style_guide/modules/_all.scss [hunk 2]
```diff
@@ -25,5 +25,7 @@
 @import "promo_banner";
 @import "select";
 @import "shame";
+@import "hero_simple";
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_all.scss]: can we keep this in alphabetical order?

Comment (alexisBimatovs) [inline app/assets/stylesheets/forever_style_guide/modules/_all.scss]: screen shot 2016-07-26 at 2 31 33 pm

**Comment by schweitzdesign** [comment]
Looks good!

**Pull request merged**